frontend/plan/components/search/SchoolReq.tsx

```diff
@@ -1,24 +1,43 @@
 import React, { useState } from "react";
 import PropTypes from "prop-types";
+import { FilterData, School, Requirement } from "../../types";
+
+interface SchoolReqProps {
+    startSearch: (searchObj: object) => void;
+    filterData: FilterData;
+    schoolReq: { [K in School]: Requirement[] };
+    addSchoolReq: (s: string) => void;
+    remSchoolReq: (s: string) => void;
+}
+
+type SchoolDisplay = "College" | "Engineering" | "Nursing" | "Wharton";
```

Comment:
Is there an enum that could be used for this type? would help with the repitition between here and the actual list.